### PR TITLE
breaking: pt: remove data stat from model init

### PR DESCRIPTION
**Pull request by iProzd**
This PR:
- clean up the data stat process from model init.
- fix import error and skipped uts.

Please note that this code PR is just an initial cleanup and refinement of the data stat, and a more detailed design of the data stat will be completed in the next PR.

### CODE REVIEW

File: deepmd/pt/model/descriptor/descriptor.py [hunk 1]
```diff
@@ -57,11 +57,19 @@ class SomeDescript(Descriptor):
...
 
+    @classmethod
+    def get_data_stat_key(cls, config):
+        """Get the keys for the data statistic of the descriptor."""
+        descrpt_type = config["type"]
+        return Descriptor.__plugins.plugins[descrpt_type].get_data_stat_key(config)
+
     @classmethod
     def get_data_process_key(cls, config):
+        """Get the keys for the data preprocess."""
```
Comment (wanghan-iapcm) [inline deepmd/pt/model/descriptor/descriptor.py:72]: one would not understand the method from such a doc str.

File: deepmd/pt/model/descriptor/descriptor.py [hunk 2]
```diff
@@ -57,11 +57,19 @@ class SomeDescript(Descriptor):
 
     @classmethod
     def get_stat_name(cls, config):
+        """Get the name for the statistic file of the descriptor."""
         descrpt_type = config["type"]
         return Descriptor.__plugins.plugins[descrpt_type].get_stat_name(config)
 
+    @classmethod
+    def get_data_stat_key(cls, config):
+        """Get the keys for the data statistic of the descriptor."""
```
Comment (wanghan-iapcm) [inline deepmd/pt/model/descriptor/descriptor.py:66]: one would not understand the method from such a doc str.

File: deepmd/pt/model/model/model.py [hunk 3]
```diff
@@ -22,127 +22,117 @@ def __init__(self):
...
-
-                energy = [item["energy"] for item in sampled]
-                mixed_type = "real_natoms_vec" in sampled[0]
-                if mixed_type:
-                    input_natoms = [item["real_natoms_vec"] for item in sampled]
+        bias_atom_e = None
+        if sampled is not None:  # compute stat
+            for sys in sampled:
+                dict_to_device(sys)
+            sumr, suma, sumn, sumr2, suma2 = self.descriptor.compute_input_stats(
```
Comment (wanghan-iapcm) [inline deepmd/pt/model/model/model.py:34]: not all the models have the data member `self.descriptor`, e.g. the pair_table atomic model. 
we should mv the method out of the base model. 

The same is for fitting. but not that the atomic_model may need the output stat.

File: deepmd/pt/model/model/model.py
Comment (wanghan-iapcm) [inline deepmd/pt/model/model/model.py:1]: you have a lot of changes on the file. please tell what have you done

**Review by wanghan-iapcm — CHANGES_REQUESTED** [review]

**Comment by iProzd** [comment]
See #3245 for further information.

**Pull request closed without merging**